src/main/java/edu/harvard/iq/dataverse/Dataset.java

```diff
@@ -423,12 +424,15 @@ public void addFileCategory(DataFileCategory category) {
         // (TODO: ? - provide these as constants somewhere? -- L.A. beta15)
         if (!ret.contains("Documentation")) {
             ret.add("Documentation");
+            //ret.add(BundleUtil.getStringFromBundle("dataset.category.documentation"));
         }
         if (!ret.contains("Data")) {
             ret.add("Data");
+            //ret.add(BundleUtil.getStringFromBundle("dataset.category.data"));
         }
         if (!ret.contains("Code")) {
             ret.add("Code");
+            //ret.add(BundleUtil.getStringFromBundle("dataset.category.code"));
```

Comment:
What's the plan for "Documentation", "Data", and "Code"? Put them in Bundle.properties as the new comments suggest? If so we can remove the TODO about constants.

Reply:
We can move them to bundle.properties, and the "if condition" should also be changed like
if(!ret.contains(BundleUtil.getStringFromBundle("dataset.category.documentation")))
{
}

Wondering whether it will affect the application logic in any other places?

Reply:
Hmm. I don't *think* we want English in there as "Data" or whatever. I'm also wondering why you just removed the comments in 30a0286 because I thought instead you might remove "Required", "Hidden", and "Optional" from being hard coded. Let me try to get some other opinions on this.

Reply:
Thanks @pdurbin and @oscardssmith .  Please confirm about 
"Required", "Hidden", and "Optional" from DataversePage.java
"Documentation" , "Data" and "Code" from Dataset.java

Reply:
@JayanthyChengan ok I just talked to @sekmiller about "Documentation", "Data", and "Code" above and "Required", "Hidden", and "Optional" in 30a0286 and in all cases, it would be best to remove these English strings from Java and put the English strings in Bundle.properties instead. Do you mind doing that?

Reply:
Thanks , I will update the programs.

Reply:
Completed.